AppAuth-JS/built/node_support/node_requestor.d.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference types="jquery" />
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork